content/influxdb/clustered/query-data/troubleshoot-and-optimize/report-query-performance-issues.md

```diff
@@ -310,3 +323,71 @@ curl --get "https://{{< influxdb/host >}}/query" \
 {{< /code-tabs-wrapper >}}
 
 {{% /code-placeholders %}}
+
+### Gather system information
```

Comment:
Is system table data local to each querier? I vaguely recall something like that being the case. If it's true, then these instructions may lead to different outputs at different times as the queries get routed to a single querier at a time. Customers may be confused when that happens, so it would be a good idea to mention in these docs (I also don't see it mentioned in `/influxdb/clustered/admin/query-system-data`).

I don't think we have a good workaround for handling this in customer-facing docs like this (i worry the instructions would need to be too complex and prone to error). In the future we could have a customer-facing tool that is aware of the querier deployment structure and can collect system information from each one-by-one to aggregate it for debug purposes.

Reply:
We try to describe it for `system.queries` in https://docs.influxdata.com/influxdb/clustered/reference/internals/system-tables/#queries-system-measurement.

@appletreeisyellow Is there a similar description for `system.tables` that I can add or should I extend this description to `system.tables`?

Reply:
> Is system table data local to each querier?

@waynr 

The system table data for `system.tables`, `system.partitions`, and `system.compactor` includes all the queriers. The data comes from the catalog, and because all the queriers share one catalog, so users will see the same results from these three tables no matter which querier they are connecting to. 

However, `system.querier` is bit different, and (like you remembered) the data from this table is local to each querier.

Reply:
> Is there a similar description for `system.tables` that I can add or should I extend this description to `system.tables`?

@jstirnaman 

The description for `system.tables`, `system.partitions`, and `system.compactor` are mentioned here https://docs.influxdata.com/influxdb/clustered/admin/query-system-data/#systemtables 

On the side note, I feel these two pages are a bit repetitive. Maybe combing them into one page?
* https://docs.influxdata.com/influxdb/clustered/reference/internals/system-tables/
* https://docs.influxdata.com/influxdb/clustered/admin/query-system-data/